3p/twitter.js

```diff
@@ -75,7 +74,7 @@ export function twitter(global, data) {
     } else if (data.timelineSourceType) {
       // Extract properties starting with 'timeline'.
       const timelineData = Object.keys(data)
-        .filter((prop) => startsWith(prop, 'timeline'))
+        .filter((prop) => prop.startsWith('timeline'))
```

Comment:
To be used here, this needs to be included in `3p/polyfills.js`

Reply:
Good to know!